src/test/java/edu/kis/powp/jobs2d/TestJobs2dPatterns.java

```diff
@@ -23,10 +26,13 @@ public class TestJobs2dPatterns {
 	 * @param application Application context.
 	 */
 	private static void setupPresetTests(Application application) {
-		SelectTestFigureOptionListener selectTestFigureOptionListener = new SelectTestFigureOptionListener(
-				DriverFeature.getDriverManager());
+		SelectTestFigureOptionListener selectTestFigureOptionListener1 = new SelectTestFigureOptionListener(
+				DriverFeature.getDriverManager(), 1);
+		SelectTestFigureOptionListener selectTestFigureOptionListener2 = new SelectTestFigureOptionListener(
+				DriverFeature.getDriverManager(), 2);
```

Comment:
You should use predefined constants:

public class SelectTestFigureOptionListener implements ActionListener {
	private final DriverManager driverManager;
	private final int figureTestNumber;
	public static final int FIGURE_1 = 1;
	public static final int FIGURE_2 = 2;